flexget/plugins/sites/argenteam.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # noqa pylint: disable=unused-import, redefined-builtin
+
+import logging
+
+from flexget import plugin
+from flexget.entry import Entry
+from flexget.event import event
+from flexget.config_schema import one_or_more
+from flexget.utils.requests import Session, TimedLimiter, RequestException
+from flexget.utils.search import normalize_scene
+from flexget.plugin import PluginError
+
+log = logging.getLogger('argenteam')
+
+class SearchArgenteam(object):
+    """ Argenteam """
+
+    schema = {
+        'type': 'object',
+        'properties': {
+            'force_subtitles': {'type': 'boolean', 'default': True},
+        },
+        "additionalProperties": False
+    }
+
+    base_url = 'http://www.argenteam.net/api/v1/'
+
+    @plugin.internet(log)
+    def search(self, task, entry, config):
+        """
+            Search for releases
+        """
+
+        entries = set()
+
+        for search_string in entry.get('search_strings', [entry['title']]):
+
+            try:
+                params = { 'q': normalize_scene(search_string) }
+                resp = task.requests.get(self.base_url + 'search', params=params)
+                log.debug('Requesting: %s', resp.url)
+                response = resp.json()
+            except RequestException as e:
+                log.error('Argenteam request failed: %s', e)
+                return
+
+            if not response:
+                log.debug('Empty response from Argenteam')
+                continue
+
+            if not response.get('total'):
+                log.debug('No results found for %s', search_string)
+            else:
```

Comment:
I missed this the first time around. This is also a redundant else-branch. You can simply put in a `continue` in the `if` and remove the else. It's a little better for readability imo.

Reply:
Ok. As I said before, I'm new on python and I'm not used to that sentence.